src/index.js

```diff
@@ -7,7 +7,7 @@ import reportWebVitals from './reportWebVitals';
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <App  data-bs-toggle="modal" data-bs-target="#exampleModal"/>
```

Comment:
exampleModal? Really? Surely we can think of a more appropriate name.

Reply:
Redundant code. Removed